src/locales/en/pages.yml

```diff
@@ -157,3 +157,9 @@ tc-amount:
   content:
     - Tell us about a tax credits payment you’ve had after {{ dynamicDate }}.
     - You’ll get tax credits payments every week or every 4 weeks. You only need to enter one payment amount.
+questionEnterRecentSelfAssessmentPayment:
```

Comment:
In future we probably want to standardise the keys in here so they all follow the `questionPageSlug` pattern if we can - I've added to a note to tech debt ticket [PYIC-6027](https://govukverify.atlassian.net/browse/PYIC-6027) as it's related